Cargo.toml

```diff
@@ -17,7 +17,9 @@ pulldown-cmark = { version = "0.9.2", default-features = false }
 pulldown-cmark-to-cmark = "10.0.4"
 regex = "1.9.4"
 semver = "1.0.16"
+serde = "1.0.130"
 serde_json = "1.0.91"
+toml = "0.5.8"
```

Comment:
Could you add a comment saying that we need to stick to the version used by `mdbook`? Because of https://github.com/rust-lang/mdBook/pull/2151. Ideally, I would like to avoid having to know this by not using `toml` directly, like we did in https://github.com/google/mdbook-i18n-helpers/pull/77#discussion_r1325560711.

However, I'm realizing that this seems like a loosing battle, so we can probably depend on it and then rely on the compiler telling us when things cannot be mixed.

Reply:
Ideally, they would re-export their version

Reply:
`.into()` did the trick x)